### PR TITLE
Enhancement: OP-48

### PR DESCRIPTION
**Pull request by Shree05Ram**
Users can change their password by clicking on the "Change Password" button present in the "Users" tab.

### CODE REVIEW

File: devmgmtui/src/components/user/UpdatePassword.js [hunk 1]
```diff
@@ -20,22 +22,41 @@ class UpdatePassword extends Component {
...
+
     handlePasswordChange(e) {
         this.setState({
             password: e.target.value
         })
     }
 
     handleSubmit() {
-        /* Handler code goes here */
+        console.log("reached handleSubmit");
```
Comment (SumukhaKV) [inline devmgmtui/src/components/user/UpdatePassword.js]: Remove this console.log()

Comment (Shree05Ram) [reply devmgmtui/src/components/user/UpdatePassword.js]: This is removed in the commit with ID c25b777

File: devmgmtui/src/components/user/UpdatePassword.js [hunk 2]
```diff
@@ -48,6 +68,12 @@ class UpdatePassword extends Component {
                                 {' '}Set a new password
                             </Header>
 
+                            <p
+                                onChange={this.handleUserChange.bind(this)}
+                                value={this.state.user} />
+
+                                <br />
+
```
Comment (SumukhaKV) [inline devmgmtui/src/components/user/UpdatePassword.js]: This tag is not required. Remove the handlers attached to it as well.

File: devmgmtui/src/components/user/UpdatePassword.js [hunk 3]
```diff
@@ -20,22 +22,40 @@ class UpdatePassword extends Component {
...
     handlePasswordChange(e) {
         this.setState({
             password: e.target.value
         })
     }
 
     handleSubmit() {
-        /* Handler code goes here */
+        if((this.state.password.length>=1) && ((this.state.password) !== ('\ '))) {
```
Comment (SumukhaKV) [inline devmgmtui/src/components/user/UpdatePassword.js]: The second condition is not required.

**Pull request merged**